src/cmd/api.js

```diff
@@ -154,6 +154,17 @@ module.exports = class ParticleApi {
 		);
 	}
 
+	listDeviceOsVersions(platformId, internalVersion, perPage=100){
```

Comment:
Since we have different params I think might worth to transform it like `{platformId, internalVersion, perPage=100} = {}` unless we can't use the params separately like just send `internalVersion` without `platformId`

Reply:
Good point. In this instance only `platformId` parameter is being used. I kept the positional arguments instead of making them named (with `{}`) to keep the function signature match the others.